software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -80,6 +81,9 @@
     public static final ConfigKey<String> OPERATION_TIMEOUT = newConfigKeyWithPrefix(BrooklynConfigKeys.BROOKLYN_WINRM_CONFIG_KEY_PREFIX, WinRmTool.OPERATION_TIMEOUT);
     public static final ConfigKey<Integer> RETRIES_OF_NETWORK_FAILURES = newConfigKeyWithPrefix(BrooklynConfigKeys.BROOKLYN_WINRM_CONFIG_KEY_PREFIX, WinRmTool.RETRIES_OF_NETWORK_FAILURES);
     public static final ConfigKey<Boolean> USE_HTTPS_WINRM = WinRmTool.USE_HTTPS_WINRM;
+    public static final ConfigKey<String> VERIFY_WINDOWS_UP = newStringConfigKey("verifyWindowsUp",
```

Comment:
OK, now I see that you are treating it as both of the above :). Still think that typing it as a `Duration` and skipping retries if null is the way to go.